lib/_element.js

```diff
@@ -355,19 +354,21 @@ function element () {
   Element.prototype.compareDocumentPosition = function (other) {
     function getFirstNodeByOrder (nodes, nodeOne, nodeTwo) {
       return nodes.reduce(function (result, node) {
-        return result !== false
-          ? result
-          : node === nodeOne
-            ? nodeOne
-            : node === nodeTwo
-              ? nodeTwo
-              : node.childNodes
-                ? getFirstNodeByOrder(node.childNodes, nodeOne, nodeTwo)
-                : false
+        if (result !== false) {
+          return result
+        } else if (node === nodeOne) {
+          return nodeOne
+        } else if (node === nodeTwo) {
+          return nodeTwo
+        } else if (node.childNodes) {
+          return getFirstNodeByOrder(node.childNodes, nodeOne, nodeTwo)
+        } else {
+          return false
```

Comment:
Although it's a lot more verbose, this style is easier to read I guess. Good move.